src/bun.js/ConsoleObject.zig

```diff
@@ -1755,16 +1752,16 @@ pub const Formatter = struct {
                     this.writer.writeAll(" ") catch unreachable;
                 }
                 if (!is_iterator) {
-                    const key = nextValue.getIndex(globalObject, 0) catch return;
-                    const value = nextValue.getIndex(globalObject, 1) catch return;
+                    const key = nextValue.getIndex(globalObject, 0) catch |err| return bun.jsc.host_fn.voidFromJSError(err, globalObject);
```

Comment:
`voidFromJSError` seems like something we should avoid. i think the `forEach` function should have a non-extern wrapper. what do you think?